libs/api/domains/driving-license-book/src/dto/createPracticalDrivingLesson.input.ts

```diff
@@ -0,0 +1,32 @@
+import { IsString, IsNumber, IsOptional } from 'class-validator'
```

Comment:
Ditton on class-validator ... not needed for GraphQL unless it's doing some complex validation, like isEmail.